canonical-websites/docs.ubuntu.com
# Remove /core docs

Remove references to /core docs as they are now served inside their own
app, which is bundled in the repository with the docs.
This is live and serving the docs on docs.ubuntu.com/core

## QA

``` bash
./run clean
./run
```

Make sure it builds and the site loads correctly. Including checking <127.0.0.1:8007/maas>

Visit <127.0.0.1:8007/core>, see that it 404s
